wp-content/themes/twentyseventeen-wp20/style.css

```diff
@@ -361,7 +361,7 @@ body.page .entry-header {
 .site-branding {
 	display: grid;
 	grid-template-rows: 1fr minmax(80px, 2fr) 1fr;
-	grid-template-columns: 1fr auto 1fr;
+	grid-template-columns: 100%;
```

Comment:
These 3 columns are set so that the main logo hit area is only the same width as the logo. Here it is working correctly in Chrome

![Screen Shot 2023-04-11 at 11 26 59 AM](https://user-images.githubusercontent.com/1017872/231018104-b1553727-4e29-416c-a48a-a8c00eed77b1.jpg)

Still trying to figure out why the column config isn't working in Firefox. For some reason the logo box doesn't seem to end at column 3.

![Screen Shot 2023-04-11 at 11 29 34 AM](https://user-images.githubusercontent.com/1017872/231018264-d0684894-bf5b-4ad9-bd45-4d0748a9ab93.jpg)